### DISCUSSION TITLE
Why are `orgId` and `projectId` gitignored?

### DISCUSSION THREAD

**Discussion by borekb** [vercel/vercel#4518] [Ideas] [answered]
From `.vercel/README.txt`:

> > Should I commit the ".vercel" folder?
>
> No, you should not share the ".vercel" folder with anyone.
> Upon creation, it will be automatically added to your ".gitignore" file.

Why are `orgId` and `projectId` gitignored? Are they different between my machine and colleagues' machines?

Wouldn't it be just easier if everyone on the team could check out the repo, `cd` into a folder and run `now` / `vercel` without the need for initial configuration?

**Comment by styfle** [comment, marked as answer]
The `.vercel` folder is automatically generated by Vercel CLI and can contain more information than just org id and project id.

> Why are orgId and projectId gitignored? 

The reason we ignore it by default is because we will store caches during `vercel dev` and perhaps other data in the future. Additionally, this would be confusing for open source projects that have contributors outside of the Vercel team/org.

> Are they different between my machine and colleagues' machines?

These two values are currently the same on all machines.

In the future, we may have a more granular `.gitignore` rule to exclude files we know are unsafe to commit but for now it is the entire directory.

**Reply by borekb** [reply]
Thanks for the explanation.
